sdk/eventhub/Microsoft.Azure.EventHubs/src/Microsoft.Azure.EventHubs.csproj

```diff
@@ -1,59 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <AssemblyTitle>Microsoft.Azure.EventHubs</AssemblyTitle>
     <Description>This is the next generation Azure Event Hubs .NET Standard client library. For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>3.0.0</Version>
-    <AssemblyVersion>3.0.0.0</AssemblyVersion>
-    <FileVersion>3.0.0.0</FileVersion>
     <VersionPrefix>3.0.0</VersionPrefix>
-    <PackageId>Microsoft.Azure.EventHubs</PackageId>
```

Comment:
Was trimming some of the metadata intentional?   These were carried over from the stand-alone project during migration.  I don't think it will cause an issue either way, but was focused on minimizing changes during migration.

Reply:
Yes I cleaned up this project and removed any redundant properties that were not needed.